pkg/ocm/ocm.go

```diff
@@ -43,19 +44,63 @@ func New(ocmConfigFile string) (Client, error) {
 	var err error
 	client := Client{}
 
-	cfg, err := newConfigFromFile(ocmConfigFile)
-	if err != nil {
-		return client, fmt.Errorf("failed to load config file: %w", err)
+	// The debug environment variable ensures that we will never use
+	// a ocm config file on a cluster deployment. The debug environment variable
+	// is only for local cadctl development
+	debugValue, hasDebug := os.LookupEnv("CAD_DEBUG")
```

Comment:
```suggestion
	debugValue, ok:= os.LookupEnv("CAD_DEBUG")
	if !ok {
	debugValue = false
	}
```

Reply:
I see, but we also have to remove the regex check then, right? Do you want to edit your suggestion or shall I just implement it?